src/nouns/Item.ts

```diff
@@ -13,6 +15,11 @@ export default class Item {
 		this.takeable = takeable;
 	}
 
+	static getInventory = (items: { [key: string]: Item }) =>
```

Comment:
So it looks like both times we use `getInventory`, we end up doing something like
```js
getInventory(items).map(({slug}) => slug)
```

Do you think `getInventory` should just do that map itself and return a list of slugs, rather than a list of items? I'm sort of torn.